.github/workflows/gh-pages.yml

```diff
@@ -1,16 +1,13 @@
 name: Deploy to GitHub Pages
 
 on:
-  pull_request:
+  push:
```

Comment:
Isnt this what caused the issue the other day?

Reply:
It's actually when it's on`pull_request` that it was giving issues

Reply:
@jmaslek please help approve. there is also a fix for nightly build here